app/views/components/gds/timeline.scala.html

```diff
@@ -0,0 +1,26 @@
+
+@import java.time.format.DateTimeFormatter.ISO_OFFSET_DATE_TIME
+
+@import views.helpers.TimelineEvent
+
+@this()
+
+@(timelineEvents: Seq[TimelineEvent])
+
+@htmlForTimelineEvent(timelineEvent: TimelineEvent) = {
+    <li class="hmrc-timeline__event">
+        <h2 class="hmrc-timeline__event-title">@timelineEvent.title</h2>
+        <time class="hmrc-timeline__event-meta" datetime="@{ISO_OFFSET_DATE_TIME.format(timelineEvent.dateTime)}">
```

Comment:
Could this be formatted the same way the displayed dateTime is formatted?

Reply:
No. It adds a semantic **timestamp** to the element, and it should be readable by screen readers, search engines, JavaScript code for apps that want to set for instance appointments, calendar remainders or something like that.